API.md

```diff
@@ -74,3 +83,11 @@ const output = diff(firstDocument, secondDocument, {
  }
 })
 ```
+<a name="Output"></a>
+
+## Output : [<code>Output</code>](#Output)
+**Kind**: global typedef  
+<a name="DiffOutputItem"></a>
+
+## DiffOutputItem : [<code>DiffOutputItem</code>](#DiffOutputItem)
+**Kind**: global typedef  
```

Comment:
Yeah, so everything else looks good. The only thing this requires is that instead of "global typedef", it should have 
```
DiffOutputItem => Array.<DiffOutput>
```

> ## DiffOutput
> action => ActionType
> path => string

I'm not too sure about the format, but it should have the information about what `DiffOutputItem` looks like. Have information about the `DiffOutput` type [here](https://github.com/asyncapi/diff/blob/e4311824893e8bb62c4e1f95544239e01ee685bf/src/types.ts#L19)

Reply:
@ron-debajyoti let me know if you need further clarification :)

Reply:
@aayushmau5  I tried to solve this but couldn't do it with jsdoc. I tried different implementations like :
- adding @typedef and @interfaces in`types.ts` and then using that in `asyncapidiff.ts`
- using typescript implementation of jsdoc [as given here](https://github.com/jsdoc2md/jsdoc-to-markdown/wiki/How-to-document-TypeScript)

None of the above solved the problem. If we are continuing to use jsdoc here, then I think this is the best we can do. 

I found out another library called TypeDoc which might solve the problem. Do you want me to explore that option ?

Reply:
Alright, so initially I was going to say No(because I don't want to take too much of your time on this issue), but I did try `typedoc` for a bit(just ran `npx typedoc src/index.ts`).

![image](https://user-images.githubusercontent.com/54525741/149777996-f7537da5-001d-400d-aa71-9c0b5a81dda0.png)

And it works like a charm! (though we have to improve something because the examples were not outputting correctly)

Anyway, I think you should give typedoc a try and see what you can come with up!(Good find, thanks @ron-debajyoti :) )

Reply:
Sounds great @aayushmau5 !  Rendering Typescript types using jsdoc2md was getting too much of a hassle, hence I just looked up alternatives. Glad this might work :smile: 

I probably will take this up tomorrow or on Wednesday as I also got another ongoing issue to solve in asyncapi/modelina. 

Do I close this and create another PR or just continue here ? I think just updating the descriptions here would be fine.

Reply:
Sure! Take your time. 
This PR is just fine, continue here.